packages/react-spinner/src/components/Spinner/useSpinnerStyles.ts

```diff
@@ -1,7 +1,14 @@
+import { SlotClassNames } from '@fluentui/react-utilities';
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { SpinnerState } from './Spinner.types';
+import type { SpinnerSlots, SpinnerState } from './Spinner.types';
 
+/**
+ * @deprecated
+ */
 export const spinnerClassName = 'fui-Spinner';
```

Comment:
For `Spinner` you can actually remove this class name instead of deprecating it since the component is not being exported from `@fluentui/react-components` yet:

```suggestion
```